README.md

```diff
@@ -10,3 +10,15 @@ A test repo to play around with automation
 - [Test workflow is described here](./test-workflow.drawio)
 - [A generic workflow to work with git is explained here](./git-workflow.drawio)
 
+# Test Automation framework
+  This repository contains information regarding test automation framework and approach. Please note that, *this repository will also be used as a playground to interact with Git as part of the automation framework.*
+  
+  As part of this framework we are mainly going to focus on the following aspects
+  - Test Environment Setup
+  - Test case setup
+  - Test data setup
+  - Configuration setup
+  - Reporting
+  - Notifications
+
+[Link to the current repository](https://github.com/PramodKumarYadav/drona).
```

Comment:
so normally when you will open a PR, people will find some feedback and they will ask you to incorporate the same. They can do it either via comments (which will not stop a PR to be merged) or via tasks. something that would need to be completed before a PR can be merged. So to learn this part, I will make a task here. 

- [ ] This line for [link to current repo] is not needed since readme is already in the repo. But I see that you did this to show that you now know how to create links - so that assignment is okay.